examples/platform/realtek/dac_provider/CommonDACPrivider.cpp

```diff
@@ -0,0 +1,43 @@
+/*
+ *
+ *    Copyright (c) 2022 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#include "CommonDACPrivider.h"
+#include <platform/realtek/BEE/RTK/RTKDACVendorProvider.h>
+#include <stddef.h>
+
+using namespace chip::DeviceLayer;
+
+static chip::Credentials::DeviceAttestationCredentialsProvider * default_dac_provider_getter(void)
```

Comment:
no need to add `(void)`. This applies throughout.

This is for https://clang.llvm.org/extra/clang-tidy/checks/modernize/redundant-void-arg.html (we can standardize on no void, cannot do the reverse)